lib/ecto/adapters/mysql/connection.ex

```diff
@@ -11,7 +11,13 @@ if Code.ensure_loaded?(Mariaex.Connection) do
 
     def connect(opts) do
       opts = Keyword.update(opts, :port, @default_port, &normalize_port/1)
-      Mariaex.Connection.start_link(opts)
+      case Mariaex.Connection.start_link(opts) do
+        {:ok, pid} ->
+          repo = Keyword.fetch!(opts, :repo)
+          repo.after_connect(pid)
```

Comment:
I am ok with doing it in a task but we shuold first check if after_connect exists in the repo, so we don't spawn 20 tasks in prod to do nothing.